### PR TITLE
Remove ceilometer-expirer cron job from the barclamp.

### PR DESCRIPTION
**Pull request by aplanas**
Move the expirer cron job to the package:

- C:OS:J https://build.opensuse.org/request/show/281290
- C:OS:M https://build.opensuse.org/request/show/281292

### CODE REVIEW

**Comment by jsuchome** [comment]
+1

**Comment by tboerger** [comment]
Does it make sense to keep that for non SUSE installations?

**Comment by dirkmueller** [comment]
+1

**Comment by aplanas** [comment]
@tboerger No idea. The expirer is more important in postgres that in mongodb (that auto-delete the TTL entries)

I guess that in non-SUSE installations, is the user who needs to manually install the cron job, as the openstack documentation describes.

**Comment by aplanas** [comment]
* Added a full path in the remaining cronjob.

File: chef/cookbooks/ceilometer/recipes/server.rb [hunk 1]
```diff
@@ -325,23 +325,6 @@
...
-    path '/etc/cron.daily/crowbar-ceilometer-expirer'
-    owner 'root'
-    group 'root'
-    mode '0755'
-    action :create
-  end
-else
-  file '/etc/cron.daily/crowbar-ceilometer-expirer' do
-    action :delete
-  end
```
Comment (vuntz) [inline chef/cookbooks/ceilometer/recipes/server.rb]: You should probably have kept that, as the cron job will be in stoney/icehouse and so we need to remove it in tex/juno.

Comment (aplanas) [inline chef/cookbooks/ceilometer/recipes/server.rb]: Makes sense to me. I will submit a new PR.

Comment (aplanas) [inline chef/cookbooks/ceilometer/recipes/server.rb]: Fixed in #169

**Pull request merged**